bin/eslint-github-init.js

```diff
@@ -68,14 +71,49 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // TODO: Keep tsconfig.json and tslint.json as files in the lib and copy it over, rather then keeping it here inline.
+
+    const tsconfigPath = path.resolve(process.cwd(), 'tsconfig.json')
+    if (!fs.existsSync(tsconfigPath)) {
+      const tsconfigDefaults = {
+        compilerOptions: {
+          target: 'es5',
+          module: 'commonjs',
+          strict: true,
+          esModuleInterop: true
+        }
+      }
+      fs.writeFileSync(tsconfigPath, JSON.stringify(tsconfigDefaults, null, '  '), 'utf8')
+    }
+
+    const tslintPath = path.resolve(process.cwd(), 'tslint.json')
+    const tslintrc = fs.existsSync(tslintPath)
+      ? JSON.parse(fs.readFileSync(tslintPath, 'utf8'))
+      : {
+          defaultSeverity: 'error',
+          extends: ['tslint-config-prettier'],
+          jsRules: {},
+          rules: {},
+          rulesDirectory: []
+        }
+    if (tslintrc.extends[tslintrc.extends.length - 1] !== 'tslint-config-prettier') {
+      tslintrc.extends.push('tslint-config-prettier')
+    }
+    fs.writeFileSync(path.resolve(process.cwd(), 'tslint.json'), JSON.stringify(tslintrc, null, '  '), 'utf8')
+  }
   if (answers.react) eslintrc.extends.push('plugin:github/react')
   if (answers.relay) eslintrc.extends.push('plugin:github/relay')
 
   fs.writeFileSync(path.resolve(process.cwd(), '.eslintrc.json'), JSON.stringify(eslintrc, null, '  '), 'utf8')
 
   const prettierConfig = []
-  if (answers.flow) prettierConfig.push('/* @flow */')
+  if (answers.typeSystem === 'flow') prettierConfig.push('/* @flow */')
+
+  // TODO: Set arrow parens to true if typescript
```

Comment:
I don't think this is needed. I think `tslint-config-prettier` does this already.